spec/javascript/packages/document-capture/components/review-issues-step-spec.jsx

```diff
@@ -68,6 +68,41 @@ describe('document-capture/components/review-issues-step', () => {
       }),
     ).to.exist();
   });
+  it('renders initially with warning page and displays attempts remaining with IPP', () => {
+    const { getByRole, getByText } = render(
+      <InPersonContext.Provider value={{ inPersonURL: true }}>
```

Comment:
This file isn't enforced by TypeScript, but `inPersonURL` should be a string. Can we set it as one?

```suggestion
      <InPersonContext.Provider value={{ inPersonURL: '/' }}>
```